src/components/UserProfile/UserProfile.tsx

```diff
@@ -0,0 +1,45 @@
+import Avatar from '@mui/material/Avatar';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Divider from '@mui/material/Divider';
+import Typography from '@mui/material/Typography';
+import { ProfileSliceState } from '../../redux/profile/slice';
+
+export const UserProfile = ({
```

Comment:
for cases  when you have props it is handy to use FC type for component like 
```typescript 
export type UserProfileProps = {
  profile: ProfileSliceState;
};
export const UserProfile: FC<UserProfileProps> = ({ profile }) => {
```

Reply:
done